### ISSUE TITLE
Allow changing font-size in-game (and not only double or nothing)

### ISSUE THREAD

**Issue by DynCoder** [OpenTTD/OpenTTD#9277]
The font size throughout the whole game are very weird and sometimes almost unreadable (e.g. the legend under the map), but when you increase the font size to the next step the font becomes unreasonably large

## Version of OpenTTD
Latest (steam, windows 10)

## Expected result
Twice the font size (or just an actual readable font size to begin with)

## Actual result
Extremely large font size

## Steps to reproduce
Options > Font size > double

**Comment by James103** [comment]
You might want to change the following files in `openttd.cfg` to your liking:
```
[misc]
small_font = 
medium_font = 
large_font = 
mono_font = 
small_size = 0
medium_size = 0
large_size = 0
mono_size = 0
small_aa = false
medium_aa = false
large_aa = false
mono_aa = false
```

**Comment by TrueBrain** [comment]
Basically, this is a feature request to ingame being able to change font-sizes (as in, the config settings). Leaving it open as such.

**Comment by Bohaska** [comment]
what type of inputs do these values support?

**Comment by LordAro** [comment]
This has *mostly* been resolved - GUI scaling has been reworked in #10114 to allow different scaling factors, and the font/size/aa settings can now be changed in game via the console as of #10278 , (Yes, we know this isn't exactly user friendly right now)
